VISUALIZERS/DATA_STRUCTURE/ARRAY_VIEW/ARRAY_VIEW.py

```diff
@@ -0,0 +1,107 @@
+from flojoy import flojoy, DataContainer
+import plotly.graph_objects as go
+import numpy as np
+import pandas as pd
+
+CELL_SIZE = 50
+FONT_SIZE = 10
+MAX_ALLOWED_SHAPE = 10
+MIN_ALLOWED_SHAPE = 4
+v_dot = "$\\vdots$"
+d_dot = "$\\ddots$"
+l_dot = "$\\ldots$"
+
+
+def numpy_array_as_table(arr: np.ndarray, placeholder: str):
+    if arr.size > MAX_ALLOWED_SHAPE:
+        converted_type = arr.astype(object)
+        new_arr = converted_type[:MAX_ALLOWED_SHAPE]
+        new_arr[MAX_ALLOWED_SHAPE - 2] = l_dot
+    elif arr.size < MIN_ALLOWED_SHAPE:
```

Comment:
Is there a point to having a min allowed shape? Why not just display the whole array?